SingularityUI/app/templates/tasksTable/tasksActiveBody.hbs

```diff
@@ -4,15 +4,15 @@
             <thead>
                 <tr>
                     <th data-sort-attribute="taskId.id">
-                        Name
+                        Task ID
                     </th>
                     <th class="hidden-xs" data-sort-attribute="taskId.startedAt">
-                        Started
+                        Started At
                     </th>
                     <th class="hidden-xs" data-sort-attribute="host">
                         Host
                     </th>
-                    <th class="hidden-xs" data-sort-attribute="taskId.rackId">
+                    <th class="visible-lg" data-sort-attribute="taskId.rackId">
```

Comment:
If you have the bandwidth it'd be worth a quick pass over the bootstrap classes for the table columns to make sure the collapse nicely (e.g. are xs and lg good breakpoints or is sm better? What are the most important to display and what can we hide?). If I remember correctly, visible-lg is actually deprecated as of 3.2 in favor of visible-(size)-(display style)